src/modules/0.0.11/aws_s3/index.ts

```diff
@@ -16,16 +32,16 @@ export const AwsS3Module: Module2 = new Module2({
         create: async (es: Bucket[], ctx: Context) => {
           const client = await ctx.getAwsClient() as AWS;
           for (const e of es) {
-            await client.createBucket(e.name);
+            await createBucket(client.s3Client, e.name);
           }
         },
         read: async (ctx: Context, id?: string) => {
           const client = await ctx.getAwsClient() as AWS;
-          const allBuckets = await client.getBuckets();
+          const allBuckets = await getBuckets(client.s3Client);
           return allBuckets
-            .filter(b => !id || b.Name === id)
-            .filter(b => !!b.Name)
-            .map(b => {
+            .filter((b: any) => !id || b.Name === id)
+            .filter((b: any) => !!b.Name)
```

Comment:
hmm can we actually type this instead of using `any`?

Reply:
Oh, we shouldn't need that anymore. That was actually the impetus that got me to start fixing `crudBuilder`!